communities/microgalaxy/lab/tools/tool_panel_view.yml

```diff
@@ -553,286 +1254,623 @@ items:
     id: toolshed.g2.bx.psu.edu/repos/iuc/picrust/picrust_predict_metagenomes
 
 - type: section
-  id: dna_barcoding_other
-  name: DNA barcoding - other
+  id: other_metabarcoding
+  name: Other Metabarcoding Tools
   items:
   - type: tool
     id: toolshed.g2.bx.psu.edu/repos/earlhaminst/lotus2/lotus2
   - type: tool
-    id: toolshed.g2.bx.psu.edu/repos/iuc/ancombc/ancombc
+    id: toolshed.g2.bx.psu.edu/bgruening/motus/motus
 
-- id: sequence_manipulation
-  text: Sequence manipulation
+
+- id: meta_genomics_transcriptomics
+  text: Metagenomics / Metatranscriptomics
```

Comment:
Should this maybe be meta-omics?

Reply:
I would keep it Metagenomics / Metatranscriptomics. Meta-omics would include also metabarcoding, metaproteomics which are in dedicated sections